api/app/src/DataFixtures/LayUserFixtures.php

```diff
@@ -149,6 +161,11 @@ class LayUserFixtures extends AbstractDataFixture
     ];
 
     private Deputy $deputy;
+    private ?Client $client2 = null;
```

Comment:
I think I would prefer it if we stuck to how the code was previously, where entities aren't stored in member variables. We can just use local variables for these, and pass them between methods where needed. Where you are adding new variables, these can just be appended to the existing method signatures.